docs/components/text-input.md

```diff
@@ -1,14 +1,14 @@
-# Date input
+## Date input
```

Comment:
Sorry, I wasn't clear. The heading levels were correct so you should put them back. I was referring to the h1 saying 'Date input' even though it's about the 'Text input' component